### PR TITLE
#1968 Impove code coverage

### PR DESCRIPTION
**Pull request by progsmile**
Remove unused code
Add statuses to person schema and update code accordingly
Improve coveralls:
- update person api tests
- add files to coveralls ignore

### CODE REVIEW

File: lib/mpi/web/views/person_view.ex [hunk 1]
```diff
@@ -10,36 +10,6 @@ defmodule MPI.Web.PersonView do
     render_many(persons, __MODULE__, "person.json", as: :person)
   end
 
-  def render("persons.json", %{persons: persons, search_params: search_params}) do
```
Comment (AlexKovalevych) [inline lib/mpi/web/views/person_view.ex]: Why it was removed, not used?

Comment (progsmile) [reply lib/mpi/web/views/person_view.ex]: @AlexKovalevych yes, `search_params` field is not passed from controller

File: lib/mpi/web/controllers/person_controller.ex [hunk 2]
```diff
@@ -11,6 +11,8 @@ defmodule MPI.Web.PersonController do
 
   action_fallback(MPI.Web.FallbackController)
 
+  @person_status_active Person.status(:active)
```
Comment (Troush) [inline lib/mpi/web/controllers/person_controller.ex]: I would not use the specific module attribute.
Just pass `Person.status(:active)` to function call.

This relates for all such patterns in this PR.

Comment (AlexKovalevych) [reply lib/mpi/web/controllers/person_controller.ex]: It won't work in pattern matching

Comment (Troush) [reply lib/mpi/web/controllers/person_controller.ex]: Ok, not thought about this.

File: test/unit/models/persons/persons_api_test.exs [hunk 3]
```diff
@@ -0,0 +1,61 @@
+defmodule MPI.PersonTest do
```
Comment (PavelVesnin) [inline test/unit/models/persons/persons_api_test.exs]: its better rename to `MPI.Persons.PersonsAPITest`

**Pull request merged**